net/udpxy/patches/001-fix_uninitialized_source_address.patch

```diff
@@ -0,0 +1,20 @@
+--- a/chipmunk/udpxrec.c
++++ b/chipmunk/udpxrec.c
```

Comment:
1. Please backport this patch as is from https://patch-diff.githubusercontent.com/raw/pcherenkov/udpxy/pull/32.patch it is possible that it will need to have refresh.

2. It requires to bump PKG_RELEASE in the Makefile.